sdk/src/claim.rs

```diff
@@ -288,6 +306,37 @@ impl Claim {
         }
     }
 
+    /// Create a new claim with a user supplied GUID.
+    /// user_guid: is user supplied guid conforming the C2PA spec for manifest names
+    /// claim_generator: User agent see c2pa spec for format
+    pub fn new_with_user_guid(claim_generator: &str, user_guid: &str) -> Self {
+        Claim {
+            remote_manifest: RemoteManifest::NoRemote,
+            box_prefix: "self#jumbf".to_string(),
+            root: jumbf::labels::MANIFEST_STORE.to_string(),
```

Comment:
I think we should use a different label, like MANIFEST_STORE_EXTENSION to refer to the extension, as this one is used for the internal c2pa label use case.

Reply:
I had that :) it got removed somewhere along the way.

Reply:
Someone tell me where a label like that should live now.

Reply:
I don't think we have a place for it. I'd just define it at the top of the file for now. Eventually we should add a utils.format file that defines the things that deal with file formats and extensions. That tends to be scattered through the code now, and is getting more complicated as we add types.